amd/src/mention_users.js

```diff
@@ -32,7 +32,11 @@ define(['jquery', 'local_mention_users/tribute'], function($) {
   module.init = function() {
 
     var reply_id = $('input[name=reply]').val();
-    var forum_id = $('input[name=forum]').val();
+    if ($('input[name=forum]').length > 0) {
+      var forum_id = $('input[name=forum]').val();
+    }else{
+      var forum_id = getParameterByName('d');
```

Comment:
Are you sure the d is the forum id? I would guess it would be the discussion id but I could easily be wrong.